plugins/org.eclipse.n4js.xtext.ide/src/org/eclipse/n4js/xtext/ide/server/XLanguageServerImpl.java

```diff
@@ -353,9 +353,8 @@ protected ServerCapabilities createServerCapabilities(InitializeParams params) {
 			executeCommandOptions.setCommands(commandRegistry.getCommands());
 			serverCapabilities.setExecuteCommandProvider(executeCommandOptions);
 		}
-		semanticHighlightingRegistry.initialize(allLanguages, clientCapabilities, client);
-		serverCapabilities.setSemanticHighlighting(new SemanticHighlightingServerCapabilities(
-				semanticHighlightingRegistry.getAllScopes()));
+
+		// serverCapabilities.setSemanticTokensProvider(new SemanticTokensWithRegistrationOptions()); // TODO
```

Comment:
What about this to-do?

Reply:
I kept it in there to find that location later faster when we implement the semantic syntax highlighting. Do you want me to remove that line?

Reply:
Ah ok, it just looked like something that was forgotten. Do we already have a ticket for the semantic syntax highlighting? If so, you could add the ticket ID here.

Reply:
I check that!